misk/src/main/kotlin/misk/hibernate/HibernateModule.kt

```diff
@@ -77,5 +78,16 @@ class HibernateModule(
     }).asSingleton()
 
     bind(Query.Factory::class.java).to(ReflectionQuery.Factory::class.java)
+
+    install(object : HibernateEntityModule(qualifier) {
+      override fun configureHibernate() {
+        bindListener(EventType.PRE_INSERT).to(TimestampListener::class.java)
+        bindListener(EventType.PRE_UPDATE).to(TimestampListener::class.java)
+      }
+    })
   }
+
+  override fun equals(other: Any?) = other is HibernateModule && other.qualifier == qualifier
```

Comment:
this is to allow multiple install calls? why does that happen?

Reply:
I added it when I was trying to diagnose why I was getting Hibernate errors. But I totally don’t need it. I’ll remove.